packages/@webex/test-helper-mock-webex/src/index.js

```diff
@@ -44,6 +45,10 @@ function makeWebex(options) {
   const internalChildren = {};
   const externalChildren = {};
 
+  const nonInternalPlugins = options.nonInternalPlugins
+    ? [...nonInternalPluginNames, options.nonInternalPlugins]
+    : nonInternalPluginNames;
+
```

Comment:
I do not see a change where the `options` send out `nonInternalPlugins`

Why do we have this change here?

Reply:
Oops, I was thinking of updating this function to add support for adding internal and public plugins on the fly but that is not needed. Removed.